cli/agent.go

```diff
@@ -278,8 +278,13 @@ func (r *RootCmd) workspaceAgent() *clibase.Cmd {
 				subsystems = append(subsystems, subsystem)
 			}
 
-			procTicker := time.NewTicker(time.Second)
-			defer procTicker.Stop()
+			environmentVariables := map[string]string{
+				"GIT_ASKPASS": executablePath,
+			}
+			if v, ok := os.LookupEnv(agent.EnvProcPrioMgmt); ok {
```

Comment:
I see that this `os.LookupEnv(agent.EnvProcPrioMgmt)` appears a few times. I'm wondering if we should cover it with a dedicated function.

Reply:
It's only really used here, in the other place we check the map passed to the agent which is better for tests, so I don't think this needs any change at the moment.

Reply:
My only concern is the future when somebody decides to introduce another variable and forgets to correct it in the test. Anyway, let's assume it is sort of nitpick 👍